search-service/src/test/kotlin/com/egm/stellio/search/web/EntityHandlerTests.kt

```diff
@@ -814,6 +814,66 @@ class EntityHandlerTests {
             )
     }
 
+    @Test
```

Comment:
for such a feature, only add tests where the business logic is implemented (in `ExpandedEntity` ?).